client/src/component/Button/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+
+const IconLabeledButton = ({ variant, color, icon, text, event, disable }) => (
```

Comment:
Button Size is missing, please find a good way to handle it?

Reply:
it will be handled in the style object that i just provided